src/TagDependencyTrait.php

```diff
@@ -161,22 +186,21 @@ public static function loadModel(
      * Invalidate model tags.
      * @return bool
      */
-    public function invalidateTags()
+    public function invalidateTags($event)
```

Comment:
This line breaks a backward compatible for users who using it in own application.